page/sass/commit_funds.scss

```diff
@@ -0,0 +1,19 @@
+.section.commit-funds{
+    h3{
```

Comment:
we are redefining font sizes like in ten places. I think we need to refactor it when all sections are done.